### ISSUE TITLE
Windows 10 1803 - Smart Screen

### ISSUE THREAD

**Issue by CMDRPhaedra** [msarilar/EDEngineer#371]
It appears that with windows 10 1803 - Smart Screen blocks the app from working... the Warn setting in Check apps and Files  doesn't seem to warn and blocks instead. It will work with check apps and files set to off. Not sure if its an issue with EDEngineer or windows 10 1803 but just thought i'd let you know.

**Comment by CMDRPhaedra** [comment]
yeah but after you click run anyway the app doesn't open on 1803...

**Comment by msarilar** [comment]
oh right I see you're talking about redstone 5 update

well I'm not sure what could be wrong (haven't got it yet) but thanks for reporting

do you have that warning at each startup? or only on update/install?

**Comment by CMDRPhaedra** [comment]
only open update / install. after the first time every time you try and launch the app it goes to look for update then nothing happens.

**Comment by msarilar** [comment]
Apparently the update has been postponed because of a bug that could be related : https://www.windowscentral.com/microsoft-holds-releasing-windows-10-version-1803-public

I guess you got the update because you opted-in the insider program? Anyway we'll see when the update comes out.

**Comment by Mattpenfold** [comment]
OK, I have managed to get the app to run. The short cut the install process creates does not work. However, I found the EDEngineer exe is this folder "C:\Users\matt\AppData\Local\Apps\2.0\J74WTOH4.YDK\CX0945YH.9NL\eden..tion_b9c6c2d0b4f2eae5_0001.0001_818e7c7fbb21a123"

When I run that exe things work as expected (although since I have not tested exhaustive I cannot rule a problem still lurks.

**Comment by Mattpenfold** [comment]
Confirmed that Build 1803 was the issue. I just rolled back Windows, and now ED Engineers installs and runs without a problem.

**Comment by msarilar** [comment]
putting the smartscreen to off seems to fix the problem although it's not very good... if at least it showed a warning like it says it does, it would be good enough.

![image](https://user-images.githubusercontent.com/5569959/39471911-210367d8-4d46-11e8-9726-d962061a548b.png)

**Comment by msarilar** [comment]
more details by @MrVaad 
https://github.com/msarilar/EDEngineer/issues/390#issuecomment-386174152

**Comment by MightyBOBcnc** [comment]
Can confirm ED Engineer would not launch in the April Update (build 17134.rs4_release.180410-1804) until after I set SmartScreen to "Off" ("Warn" was not actually warning; it just silently prevented the app from launching).

**Comment by jimbo5678** [comment]
confirmed here, but... you can turn smartscreen off, run the app once, making sure you uncheck "ask every time" for the unable to identify publisher dialog, then once it's running, turn smartsceen back on. After that, EDEngineer will launch successfully with smartscreen still enabled

**Comment by Sparfell-git** [comment]
Hi

Even after disabling smartscreen on an updated Win 10, EDenginner update seems to be stopped and EDenginner is not warking anymore.

**Comment by Pootch** [comment]
Hello.
My EdEngineer also does not start anymore from the shortcut... works only starting the exe directly Windows 10 1803...

**Comment by TheDarchy** [comment]
Can confirm that **jimbo5678** method works for me.

**Comment by ethzero** [comment]
Also can confirm that @jimbo5678 's method works for me too.
Windows 10 Pro N, v1803 (Build 17134.48)

**Comment by RangerGH** [comment]
I downloaded the app yesterday on the latest version of Windows 10 - just installed and updated and had the same issue.  I copied the files to my desired location, ran the exe and it worked.

**Comment by BorisHaeussler** [comment]
I have a similar problem, but can not solve it this way.
I wanted to install the newest version. (Not RUN it, at least the old version runs just fine, but INSTALL it).
Windows blocks me from doing this and even switching SmartScreen/OneClick and all other security measures OFF does NOT permit me to install this app.
I was just on the Microsoft help lime for 45 minutes and he was as confused as I am and wasn't able to solve the issue.

The problem is escalated, I am waiting for a callback on Saturday morning (I only found this thread afterwards) and can report if they find anything.
